tests/unit/auth/unauthenticated.ts

```diff
@@ -0,0 +1,83 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import nock from 'nock';
```

Comment:
Would it also make sense to adapt the `it('should register a new user', async () => {` test because that is the endpoint that will always use this client?